edit/service.vue

```diff
@@ -295,22 +321,23 @@ export default {
     :resource="value"
     :selected-subtype="serviceType"
     :subtypes="defaultServiceTypes"
-    :validation-passed="true"
-    :errors="errors"
+    :validation-passed="formIsValid"
+    :errors="unboundValidationErrors"
     :apply-hooks="applyHooks"
     @error="(e) => (errors = e)"
     @finish="save"
     @cancel="done"
     @select-type="(st) => (serviceType = st)"
   >
-    <NameNsDescription v-if="!isView" :value="value" :mode="mode" />
+    <NameNsDescription v-if="!isView" :value="value" :mode="mode" :text-rules="getAndBindPathRules('metadata.name')" />
```

Comment:
to me the function name `getAndBindPathRules` is a little confusing. Is it basically mapping rules to the YAML value they apply to? If so maybe there could be a function called `mapRulesToField` or something like that.